src/controller/CHIPDeviceController.cpp

```diff
@@ -289,12 +289,48 @@ void DeviceController::ReleaseOperationalDevice(NodeId remoteDeviceId)
     mCASESessionManager->ReleaseSession(mFabricInfo->GetPeerIdForNode(remoteDeviceId));
 }
 
+CHIP_ERROR DeviceController::CloseSession(NodeId nodeId)
```

Comment:
Possibly for a follow up, but I think we need to think through how the stop commands work in general - we have close session now for case, but stop pairing for pase as well as unpair, which does nothing. I think we should probably consolidate (not necessarily in this PR).

Reply:
Added TODO.